libs/service-portal/settings/access-control/src/screens/AccessControl/components/Accesses/Accesses.tsx

```diff
@@ -70,17 +70,19 @@ function Accesses(): JSX.Element {
                 <EmptyImage width="100%" />
               </Box>
             ) : (
-              authDelegations.map((delegation) => (
-                <AccessCard
-                  key={delegation.id}
-                  title={delegation.to.name}
-                  validTo={delegation.validTo}
-                  description={kennitala.format(delegation.to.nationalId)}
-                  tags={delegation.scopes.map((scope) => scope.displayName)}
-                  href={`${pathname}/${delegation.id}`}
-                  group="Ísland.is"
-                />
-              ))
+              authDelegations.map((delegation) =>
+                delegation.to ? (
```

Comment:
We should think about how we should handle if the `to` object is not found in NationalRegistry.

Reply:
I assume this is only for data that's on dev (before we hooked up the national registry if I remember correctly)... don't see this ever happening on prod.